Pipelines/ci.yaml

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# CI build producing developer packages.
+
+variables:
+- template: Config/settings.yaml
+- name: Version.Revision
+  value: $[counter(variables['MRTKGraphicsToolsVersion'], 1)]
+- name: Version.PreReleaseRevision
+  value: $[counter(variables['MRTKGraphicsToolsReleaseTag'], 1)]
+
+resources:
+  repositories:
+  - repository: PipelineTools
+    type: git
+    endpoint: ToolsAccess
+    name: tools.internal
+    ref: 78ecf591964ecf511b4c7a0cd84d98ce11e8be16
+
+pr: none # disable PR validation
+trigger:
+  tags:
+    include:
+    - '*'
+  branches:
+    exclude:
+    - main
+    - user-tayomadein-ci-cd
```

Comment:
Is the idea here to _only_ run this when a tag is created? Or is there also the goal of running on every merge or something?

I'm mostly wondering about the explicit exclusion of `main`. Is there a specific branch that's used for releases?

Reply:
Yes, the goal here is to run the pipeline only when a tag is created, running on every merge to main will make the publish task for the job fail and it will be failing because the package name will already exist in the pipeline.

Sample: https://dev.azure.com/EDT/MRTK/_build/results?buildId=78387&view=logs&j=00a17365-9fc3-5f6d-b77f-ab9491956439&t=6657bc81-5a29-5f52-a71a-7b7eb826b852&l=24

Currently, main is used for releases, I am looking to include a flow that only recognizes tags as the release. What do you think about this approach.